api/catalog/api/examples/audio_responses.py

```diff
@@ -36,6 +36,10 @@
     "related_url": f"{origin}/v1/audio/{identifier}/related/",
     "category": "music",
     "duration": 270000,
+    "bit_rate": 128000,
+    "sample_rate": 44100,
+    "filesize": "7139840",
```

Comment:
So great that we can add `filesize` and `filetype` here!
Why is filesize a string, though?

Reply:
It's stored as a number in ES, it's just a serializer bug that makes it a string when sending the response. #696 fixes this by removing the manual definition altogether.